lib/runners/processor.rb

```diff
@@ -177,18 +177,35 @@ def add_warning_if_deprecated_version(minimum:, file: nil, deadline: nil)
       end
     end
 
-    def add_warning_if_deprecated_options(keys)
-      deprecated_keys = config_linter.slice(*keys).compact.keys
-        .map { |k| "- `#{config_field_path(k)}`" }
+    def add_warning_if_deprecated_options
+      name = :options
+      return unless config_linter[name]
 
-      unless deprecated_keys.empty?
-        add_warning <<~MSG, file: config.path_name
-          DEPRECATION WARNING!!!
-          The following options in your `#{config.path_name}` are deprecated and will be removed.
-          See #{analyzer_doc} for details.
-          #{deprecated_keys.join("\n")}
-        MSG
-      end
+      file = config.path_name
+      add_warning <<~MSG, file: file
+        DEPRECATION WARNING!!!
+        The `#{config_field_path(name)}` option is deprecated in your `#{file}`. Fix it as follows:
```

Comment:
How about this? I think the option is deprecated for Sider in whatever file.

```suggestion
        The `#{config_field_path(name)}` option is deprecated. Fix your `#{file}` as follows:
```

Reply:
Nice idea! 👍